cdk/bin/cdk.test.ts

```diff
@@ -0,0 +1,10 @@
+import {riffRaff} from "./cdk";
+
+describe('The riff-raff output YAML', () => {
+  it('matches the snapshot', () => {
+    // @ts-ignore
+    const outdir = riffRaff.outdir; // this changes for every test execution and best not to change cdk.ts too much
+    const riffRaffYaml = riffRaff.toYAML().replaceAll(outdir, 'cdk.out');
```

Comment:
minor: Might be able to set an [env var](https://github.com/aws/aws-cdk/blob/04e5480399c2ff3f3487583343fd3021a63b63e1/packages/aws-cdk-lib/cx-api/lib/cxapi.ts#L1-L4) instead.